egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -535,7 +535,10 @@ def smooth_presoftmax_prior_scale_vector(pdf_counts,
     return scaled_counts
 
 
-def prepare_initial_network(dir, run_opts, srand=-3):
+def prepare_initial_network(dir, run_opts, srand=-3, input_model=None):
+    if input_model is not None:
+        shutil.copy2(input_model, "{0}/0.raw".format(dir))
```

Comment:
it seems strange to me to use copy2 (which copies timestamps etc.)  I think this should just be 'copy'.

Reply:
I referred to [this](https://github.com/naxingyu/kaldi/blob/master/egs/wsj/s5/steps/libs/nnet3/train/common.py#L323) line of code. Should I change both?